lib/cloud/aws/policy.go

```diff
@@ -496,6 +498,20 @@ func matchTag(policyTags []*iam.Tag, name, value string) bool {
 	return false
 }
 
+// PolicyARN returns the ARN representation of an AWS IAM Policy.
+func PolicyARN(partition, accountID, policy string) string {
```

Comment:
Nit: maybe a good place to put these functions is [`lib/utils/aws/aws.go`](https://github.com/gravitational/teleport/blob/master/lib/utils/aws/aws.go). There are some similar functions, but for the IAM role and user.

Reply:
```
	awsapiutils "github.com/gravitational/teleport/api/utils/aws"
	awslib "github.com/gravitational/teleport/lib/cloud/aws"
	awslibutils "github.com/gravitational/teleport/lib/utils/aws"
```

:hot_face: